app/src/androidTest/java/fr/free/nrw/commons/NavigationBaseActivityTest.kt

```diff
@@ -15,7 +16,7 @@ import org.junit.runner.RunWith
 @RunWith(AndroidJUnit4::class)
 class NavigationBaseActivityTest {
     @get:Rule
-    var activityRule: ActivityTestRule<*> = ActivityTestRule(AboutActivity::class.java)
+    var activityRule: ActivityTestRule<*> = ActivityTestRule(NavigationBaseActivity::class.java)
```

Comment:
NavigationBaseActivity is an abstract class, and is just inherited by others. This test now fails.

I created this test to perform tests regarding the navigation drawer, built into NavigationBaseActivity. I chose AboutActivity as it works when the user is authenticated or not, and has the nav drawer so is a good place to start. It doesn't really need to be tested for orientation changes as its individual pages should check for this. The one exception might be trying to open the navigation drawer and then rotate in case that causes a crash.

Reply:
Ohh I am sorry for showing that :(